syntax = "proto3";
package grpc;

option go_package = "grpcpb/";

message DocumentRequest {
    string number = 1;
    string format = 2;
}

message DocumentResponse {
    string number = 1;
    int32 response_code = 2;
}

service DocumentValidation {
    rpc ValidateDocument (DocumentRequest) returns (DocumentResponse);
}
